docs/decisions/0008-zbar-support-in-1.3.x.rst

```diff
@@ -0,0 +1,89 @@
+QR Scanning the native ZBar Dependency in 1.3.x
+###############################################
+
+:Date: 2023-05-22
+:Status: Completed.
+:Author: AustEcon
+
+Context
+-------
+Brief Summary
+^^^^^^^^^^^^^
+v1.3.15 has an issue with QR Scanning see: https://github.com/spesmilo/electrum/issues/6018.
+Our intention was for v1.3.16 to use the “new zbar” dll instead of the "old zbar" dll.
+After further investigation, this would (a) not fix this problem (b) require a huge amount
+of work to implement an actual fix (see details below).
+
+Detailed Explanation:
+^^^^^^^^^^^^^^^^^^^^^
+The existing 1.3.15 release of ElectrumSV uses an old version of the zbar dll
+which halted maintainance in 2012 (see: https://github.com/ZBar/ZBar ).
+
+Maintenance was taken over in a forked version (see: https://github.com/mchehab/zbar )
+and this is what most people use now including ElectrumSV's develop branch, Electrum and
+Electron-Cash. However, in all of these cases, they do not let zbar manage the image
+acquisition or accessing the camera. This is done via the PyQt6 QtCamera feature and
+the pre-processed image array data is submitted to zbar for interpretation.
+
+By letting PyQt6 manage the camera, this sidesteps an unresolved issue:
+see: https://github.com/spesmilo/electrum/issues/6018
+
+I interpret this as a bug in zbar's `dshow.c` file where pre-scaling of initial window size invalidates
+the assertion regarding buffer size.
+
+Running v1.3.15 (using "old zbar") on my windows 11 system, I can reproduce this identical error. ::
+
+    Assertion failed: img->datalen == bufferlen, file video/dshow.c, line 548
+
+Sadly, this error is also reproducible with the "new" zbar and the zbarcam.exe built-in executable.
+(See compilation instructions below).
+- Fresh install of msys2 on windows.
+- Compile with instructions from https://github.com/mchehab/zbar/blob/master/README-windows.md ::
+
+    pacman -Syu –noconfirm autoconf libtool automake make autoconf-archive pkg-config gettext-devel
+    pacman -S git
+    git clone https://github.com/mchehab/zbar.git
+    cd zbar
+    pacman -S mingw-w64-i686-gcc
+    autoreconf -vfi ./configure
+        –host=i686-w64-mingw32 –prefix=/usr/local/win32
+        –without-gtk –without-python –without-qt –without-java
+        –without-imagemagick –enable-pthread
+        –with-directshow –disable-dependency-tracking
+    make
+    make install
+
+Run the built-in zbarcam.exe with the `prescale` option set (it works without the `prescale`
+option but the resulting initial window size is absurdly large::
+
+    $ zbar\zbarcam\bin> .\zbarcam.exe --prescale=640x480
+    Assertion failed: img->datalen == bufferlen, file video/dshow.c, line 548
+
+To add insult to injury, the 1.3.x series uses a 32-bit version of python and PyQt5.
```

Comment:
Remove "to add insult to injury" and use direct clear wording. This document like any contributed to ElectrumSV should not assert the emotional opinion of the author.